src/BookSettings.ts

```diff
@@ -51,16 +61,30 @@ export default class BookSettings {
             }
             this.selectedView = selectedView;
         }
+
+        if (this.fontSizes.length >= 1) {
+            let selectedFontSize = await this.store.get(BookSettings.SELECTED_FONT_SIZE_KEY);
+            if (!selectedFontSize) {
+                const defaultFontSizeIndex = Math.floor(this.fontSizes.length / 2);
```

Comment:
I think it would be nice to be able to set the default font size explicitly rather than assume it's whatever the font size in the middle of `this.fontSizes` happens to be. Perhaps `create` can also take an `initialFontSize` argument and it can throw an exception (or default to a size in the middle) if the initial size is not within the array?